hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  ViewController.swift
+//  hw_3
+//
+//  Created by Илья Мунгалов on 04.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+             
+        /* Универсальные функции сложения, вычитания, умножения и деления (перегрузка функций, одинаковые названия для функций, разные типы параметров)*/
+        
+        summa(3, 4)
+        summa(3.4, 5)
+        summa(4, 3.5)
+        summa(23.43, 23.64)
+        
+        subtraction(5, 4)
+        subtraction(6.4, 5)
+        subtraction(6, 3.5)
+        subtraction(245.43, 23.64)
+        
+        multiply(3, 4)
+        multiply(3.4, 5)
+        multiply(4, 3.5)
+        multiply(23.43, 23.64)
+        
+        devision(5, 4)
+        devision(6.4, 5)
+        devision(6, 3.5)
+        devision(245.43, 23.64)
+        
+        // Вычислить сумму цифр четырехзначного числа
+        sumFour(number: 9999)
+        
+        // Функция сравнения строк
+        stringComparison(firstString: "absfda", secondString: "absfda")
+        
+        // Циклический вызов функций - поломать приложение
+        //errorRecursion()
+        
+        // Функция возведения в степень с дефолтным параметром
+        numberPow(5)
+        
+        // Функция вычисления факториала числа (рекурсия)
+        factorial(10)
+        
+    }
+
+    func summa(_ firstnumber: Int,_  secondNumber: Int) {
```

Comment:
`summa` is a Russian word so this can be considered as a transliteration. Please, rename it to `sum`

We also use camelCase when we name our properties. Maybe you've just misspelled but please change `firstnumber` to `firstNumber`

We always leave a space after `,`, please add it here